source/scss/components/_multioffer-teaser.scss

```diff
@@ -187,13 +184,18 @@
 
   > div > .multi-teaser-info {
     $teaser-info-margin-top: .5rem;
-    flex: 1 1 100%;
+    flex: 1 1 50%;
     height: $small-line-height + $teaser-info-margin-top;
     overflow: hidden;
 
     p {
       @extend %small;
       margin-top: $teaser-info-margin-top;
+
+      @include media-breakpoint-up(md) {
+        line-height: 1.25rem;
```

Comment:
Can you add this to the variables/_typography-variables.scss ?
At the very bottom, you can find Multioffer teaser specials